src/Mod/Mesh/Gui/Resources/Mesh.qrc

```diff
@@ -14,6 +14,7 @@
         <file>icons/Mesh_Remove_Components.svg</file>
         <file>icons/Mesh_Tree_Curvature_Plot.svg</file>
         <file>icons/MeshWorkbench.svg</file>
+		<file>icons/Mesh_Fill_up_Holes.svg</file>

```

Comment:
Looks like tabs are used

Reply:
I followed the format of other names for svg icons... with tabs...

Reply:
I don't know, indentation seems off to me, don't you see it?

Reply:
![image](https://user-images.githubusercontent.com/361985/90344034-a334ab00-e01e-11ea-9bfd-b423da3a5dd6.png)

Reply:
Realy? I used Notepad++ for edition. In my laptop looks fine... :(

Reply:
This is my first time doing a PR how should I fix this?

Reply:
Notepad++ and other editors will show the Tab character like 4 spaces, but in GitHub they are displayed like 8 spaces. Many editors have the option of converting the Tab characters to a set number of spaces, or to automatically recognize the indentation based on the current file.

Edit the files to remove the Tab, and enter spaces instead. Then make a new commit.
```
git commit -a -m "Second commit"
```

However, since your changes are simple, you can squash the two commits into one. This is described in [Source_code_management](https://wiki.freecadweb.org/Source_code_management).
```
git rebase -i @~2
```

Select the second commit and prepend with `f`, which means "squash without editing the message".

Then you can force push the new head of your branch (just one commit).
```
git push -f origin mesh-svg-icons
```

This pull request will be automatically updated with the new commit.

Reply:
![Code_6eKmETsI5l](https://user-images.githubusercontent.com/5121290/90344524-8da48e80-dfe8-11ea-8158-023b48b9e7c0.png)
 Even in Visual studio it looks fine...

Reply:
> Notepad++ and other editors will show the Tab character like 4 spaces, but in GitHub they are displayed like 8 spaces. Many editors have the option of converting the Tab characters to a set number of spaces, or to automatically recognize the indentation based on the current file.
> 
> Edit the files to remove the Tab, and enter spaces instead. Then make a new commit.
> 
> ```
> git commit -a -m "Second commit"
> ```
> 
> However, since your changes are simple, you can squash the two commits into one. This is described in [Source_code_management](https://wiki.freecadweb.org/Source_code_management).
> 
> ```
> git rebase -i @~2
> ```
> 
> Select the second commit and prepend with `f`, which means "squash without editing the message".
> 
> Then you can force push the new head of your branch (just one commit).
> 
> ```
> git push -f origin mesh-svg-icons
> ```
> 
> This pull request will be automatically updated with the new commit.

Ok. But the funny thing is I did not use tab or spaces.. I did copy-paste from the previous line...